src/coreclr/jit/gentree.cpp

```diff
@@ -26236,18 +26121,18 @@ GenTree* Compiler::gtNewSimdShuffleVariableNode(
         assert(op2DupSafe != nullptr);
 
         // get the CorInfoType used for the index comparison
-        CorInfoType corType = CORINFO_TYPE_UBYTE;
+        var_types corType = TYP_UBYTE;
```

Comment:
nit: This comment is no longer correct and the local name should probably be changed (maybe `baseType` or similar is fine, if not already used).